build/local/environment.json

```diff
@@ -0,0 +1,8 @@
+{
+  "authServiceUrl": "http://localhost:50000",
+  "mgmtServiceUrl": "http://localhost:50000",
+  "adminServiceUrl": "http://localhost:50000",
+  "subscriptionServiceUrl": "https://sub.zitadel.dev",
```

Comment:
do we need this?

Reply:
good question i check out what happens if no url is provided

Reply:
removed it